src/Microsoft.AspNetCore.Hosting/WebHostBuilderExtensions.cs

```diff
@@ -51,8 +57,15 @@ public static IWebHostBuilder UseStartup(this IWebHostBuilder hostBuilder, Type
         {
```

Comment:
What about the overload here:
https://github.com/aspnet/Hosting/blob/2de091e154cc33a5b3d413572826c523d98462a4/src/Microsoft.AspNetCore.Hosting.Abstractions/HostingAbstractionsWebHostBuilderExtensions.cs#L52-L63

Reply:
What about it? This works fine.

Reply:
https://github.com/aspnet/Hosting/pull/1580/files#diff-575646ff9c6b4006e3d4eb641c7c8f01R105

Reply:
https://github.com/aspnet/Hosting/blob/111d6932a07cb732b304e04025348deba4d2ed75/src/Microsoft.AspNetCore.Hosting/GenericHost/GenericWebHostBuilder.cs#L105